codeship-services.yml

```diff
@@ -0,0 +1,6 @@
+builder:
+  build:
+    dockerfile: deployment/Dockerfile
+    encrypted_args_file: deployment/encrypted/build_args.sec
+  cached: true
+  encrypted_env_file: deployment/encrypted/build_args.sec
```

Comment:
I just compared this to visualiser and noticed `encrypted_env_file` and `encrypted_args-file` are swapped. 🤔 Not sure if it makes any difference.

https://github.com/PropellerAero/visualiser/blob/dev/codeship-services.yml#L3

Reply:
i18n does it this way.  I think I'll have to try it...
https://github.com/PropellerAero/i18n-js/blob/master/codeship-services.yml